src/configure-apollo-network.js

```diff
@@ -10,25 +10,29 @@
  * Requests will automatically timeout after 1000ms, unless another
  * timeout is provided via a 'timeout' property.
  */
-const ApolloClient = require("apollo-client");
-const apolloCacheInmemory = require("apollo-cache-inmemory");
-const apolloLinkHttp = require("apollo-link-http");
-const fetch = require("node-fetch");
+import * as ApolloClient from "apollo-client";
+import {InMemoryCache} from "apollo-cache-inmemory";
+import {createHttpLink} from "apollo-link-http";
+import fetch from "node-fetch";
 
-const createHttpLink = apolloLinkHttp.createHttpLink;
-const InMemoryCache = apolloCacheInmemory.InMemoryCache;
+import type {RenderContext} from "./types.js";
 
 const BAD_URL = "BAD_URL";
 
-const timeout = async (timeout, errorMsg) => {
+const timeout = async (timeout: number, errorMsg: string): Promise<void> => {
     return new Promise((resolve, reject) => {
         setTimeout(() => {
             reject(new Error(errorMsg));
         }, timeout);
     });
 };
 
-const configureApolloNetwork = (context) => {
+export default function configureApolloNetwork(context: RenderContext): void {
+    const {ApolloNetwork} = context;
+    if (ApolloNetwork == null) {
+        return;
+    }
```

Comment:
This check was originally done at the callsite. I moved it here because I think it's cleaner and it made flow errors easier to get rid of.